charts/beta/vector/templates/aws-config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: v1
+data:
+  config: |
+    [default]
+    region                  = {{ .Values.vector.sources.cloudtrail.region }}
+    role_arn                = {{ .Values.vector.sources.cloudtrail.cloudtrail_role_arn }}
+    web_identity_token_file = /var/run/secrets/kubernetes.io/serviceaccount/token
+kind: ConfigMap
+metadata:
+  name: vector-cloudtrail-aws-config
+  namespace: {{ .Release.Namespace }}
```

Comment:
nit: this is a pretty strange way to write a k8s manifest. it's typical to put the kind/metadata before the `data` fields

Reply:
Also you can leave out the `namespace` portion, helm will set that to .Release.Namespace by default